### ISSUE TITLE
automatically extract learner param description from help page

### ISSUE THREAD

**Issue by berndbischl** [mlr-org/mlr#1661]

**Comment by giuseppec** [comment]
This might be helpful https://github.com/openml/openml-r/issues/276

**Comment by giuseppec** [comment]
Does anyone know something better than this:

```r
library(XML)

getLearnerParamHelp = function(fun) {
  # the R package containing 'fun' must be loaded
  html = capture.output(tools:::Rd2HTML(utils:::.getHelpFile(help(fun))))
  html = htmlParse(html)
  tab = getNodeSet(html, "//table[@summary='R argblock']")
  XML::readHTMLTable(tab[[1]], header = FALSE)
}

library(ranger)
getLearnerParamHelp("ranger")
```

**Comment by mb706** [comment]
Very nice! I'll try to add it to the [learnerHelp pr](https://github.com/mlr-org/mlr/pull/1657) later.

**Comment by mb706** [comment]
Closed by #1657
